lang/rust-bootstrap/Portfile

```diff
@@ -1,25 +1,27 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem                  1.0
-
 PortGroup   rust            1.0
+PortGroup   rust_build      1.0
 
 name                        rust-bootstrap
 
 # keep in mind that you also need to update cargo.crates at the end of this file
-set version_current         1.71.1
-set version_m1              1.70.0
+set version_current         1.75.0
+set version_m1              1.74.0
```

Comment:
As before, I do not think that `version_m1` is meaningful. I know it's "minus 1", but I think there should be a better name. In this case, `version_m1` would be the `stage0` version.

Reply:
I removed `${version_m1}`.